PermanentOrg/web-app
# Per 9348 storage bar does not update after upload or deletion

This fixes the issue when a user uploads or deletes a file and the account storage bar does not update.

Steps to test: 

1.Upload/Delete any file folder.
2. Check the storage bar

@meisekimiu you might wonder why the request here: 
`getFileOrFolderSize(): number {
    let size = 0;
    if (this.item instanceof FolderVO) {
      this.api.folder.get([this.item]).then((response: FolderResponse) => {
        const folder = response.getFolderVO();
        size = folder.FolderSizeVO.allFileSizeDeep;
      });
    } else {
      size = this.item.size;
    }
    return size;
  }`

For some reason the get all function that retrieves all the items does not populate the FolderSizeVO inside a folder, only this request does ( as far as I have found out), so this was the only workaround I could have think of. If you have any other ideas in mind, please let me know!
